docs/self-managed/setup/guides/add-extra-manifests.md

```diff
@@ -0,0 +1,39 @@
+---
+id: add-extra-manifests
+title: "Add extra manifests to the Camunda 8 Helm chart"
+sidebar_label: "Add extra manifests"
```

Comment:
I'm wondering if "Add Kubernetes manifests" would be better in both of these locations. When finding this page in the sidebar, would that be more descriptive (and give an indication that it's deployment-specific)? It might also help with searching for this page.

Reply:
Changed both to "Add Kubernetes manifests"